doc/source/examples/incompressible-flow/2d-mms/2d-mms.rst

```diff
@@ -0,0 +1,251 @@
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~
+
+The velocity components are set to zero on all of the boundary to match the analytical solution. However, the condition on each boundary can be changed depending on the analytical function used.
+
+.. code-block:: text
+
+  subsection boundary conditions
+  set number = 1
+    subsection bc 0
+      set id = 0
+      set type = noslip
+    end
+  end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~
+
+For this analysis, the Reynolds number (based on the maximum velocity in the domain) is taken to be 1, which results in the terms of the NS equation being of the same order. Hence, the kinematic viscosity is set to 1 in the ``physical properties`` subsection:
+
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 1
+      end
+    end
+
+
+FEM Interpolation
+~~~~~~~~~~~~~~~~~
+
+Similarly to the ``initial refinement`` parameter in the case of a simplex mesh, ``velocity order`` and ``pressure order``  are set to the {{Poly_deg_u}} and {{Poly_deg_p}} parameter variables, respectively, in Jinja2 format.
+
+.. code-block:: text
+
+  subsection FEM
+    set velocity order = {{Poly_deg_u}}
+    set pressure order = {{Poly_deg_p}}
+  end
+
+Non-linear Solver
+~~~~~~~~~~~~~~~~~
+
+The nonlinear solver's tolerance is set to :math:`10^{-10}` since the errors on the velocity and pressure fields reach values that are near or lower than the default tolerance of :math:`10^{-8}` on the quadrilateral meshes with a finer resolution. Hence, if the default tolerance were used, the error would be constrained by this value.
+
+.. code-block:: text
+
+    subsection non-linear solver
+      subsection fluid dynamics
+        set tolerance = 1e-10
+        set verbosity = verbose
+      end
+    end
+
+
+Linear Solver
+~~~~~~~~~~~~~
+
+The only modification made in the linear solver section is the use of AMG preconditioner to accelerate the simulations. 
+
+.. code-block:: text
+
+  subsection linear solver
+    subsection fluid dynamics
+      # Set type of preconditioner for the iterative solver
+      set preconditioner = amg
+      set verbosity = verbose
+    end
+  end
+
+-----------------------
+Running the Simulations
+-----------------------
+
+
+The simulations are launched by first running the ``generate_cases.py``, then ``launch_cases.py`` scripts. The first script generates the folders and parameter files for the different configurations simulated,
+while the second one launches the simulations.
```

Comment:
I would also give an indication of the expected run time

Reply:
Your last Q3 simulations are very memory intensive, No need to change it but it's a bit intense for someone that would be running this one hteir laptop for example. I think that if you don,t have 16Gig of ram they would crash

Reply:
Then runtime is quite long for what you are trying to achieve. If I  would reduce the number of refinement to 4 so that the runtime of the simulations is much shorter.  I don,t think the example needs to such a long run time and the last point for quads don't add anything to the analysis

Reply:
(for example, some of the simulations take up to 10 minutes, that' s  a bit too much for what you are trying to demonstrate here :)